.cirrus.yml

```diff
@@ -1,3 +1,12 @@
+env:
+  # see https://github.com/SonarSource/re-terraform-aws-vault/blob/master/orders/bubble-cfamily.yaml
+  CIRRUS_VAULT_URL: https://vault.sonar.build:8200
```

Comment:
This repository is public. Is it still fine to have this URL unmasked?

Reply:
yes it's like that for all public repos